### PR TITLE
chore: Upgrade - Material Sample to run with latest uno and latest VS…

### PR DESCRIPTION
**Pull request by aesteves900**
… 2022

### CODE REVIEW

**Comment by aesteves900** [comment]
@jeromelaban , Just for the Gtk project BindableMetadataProvider doesn't autogenerate for Uno. 

Do you know why BindableMetadataProvider does not generate just for Gtk?

image

**Comment by jeromelaban** [comment]
> @jeromelaban , Just for the Gtk project BindableMetadataProvider doesn't autogenerate for Uno.
> 
> Do you know why BindableMetadataProvider does not generate just for Gtk?
> 
> image

This generally means that the Gtk head is not setup properly (missing `Runtime.Gtk` packages or similar).

File: UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj [hunk 1]
```diff
@@ -14,15 +14,29 @@
     <UpToDateCheckInput Include="..\UnoMaterialSample.Shared\**\*.xaml" />
   </ItemGroup>
   <ItemGroup>
-	<PackageReference Include="Uno.Material" Version="1.3.1" />
-	<PackageReference Include="Uno.Toolkit.UI" Version="1.3.0" />
-    <PackageReference Include="Uno.Toolkit.UI.Material" Version="1.3.0" />
-    <PackageReference Include="Uno.UI" Version="4.1.9" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.1.9" />
```
Comment (jeromelaban) [inline UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj:23]: You're missing this one in the updated project.

Comment (aesteves900) [reply UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj:23]: Still in the same issue after the update. I'm checking.


image

Comment (jeromelaban) [reply UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj:23]: I'm not seeing this particular package in the changeset, make sure it's there.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**